pandas/tests/io/pytables/test_store.py

```diff
@@ -79,33 +77,31 @@ def test_format_kwarg_in_constructor(self, setup_path):
 
         msg = "format is not a defined argument for HDFStore"
 
-        with ensure_clean_path(setup_path) as path:
+        with tm.ensure_clean(setup_path) as path:
             with pytest.raises(ValueError, match=msg):
                 HDFStore(path, format="table")
 
     def test_context(self, setup_path):
-        path = create_tempfile(setup_path)
-        try:
-            with HDFStore(path) as tbl:
-                raise ValueError("blah")
-        except ValueError:
-            pass
-        finally:
-            safe_remove(path)
-
-        try:
-            with HDFStore(path) as tbl:
-                tbl["a"] = tm.makeDataFrame()
-
-            with HDFStore(path) as tbl:
-                assert len(tbl) == 1
-                assert type(tbl["a"]) == DataFrame
-        finally:
-            safe_remove(path)
+        with tm.ensure_clean(setup_path) as path:
+            try:
+                with HDFStore(path) as tbl:
+                    raise ValueError("blah")
+            except ValueError:
+                print("Context error")
+            try:
+                with HDFStore(path) as tbl:
+                    tbl["a"] = tm.makeDataFrame()
+            except ValueError:
+                print("Context error")
+            try:
+                with HDFStore(path) as tbl:
+                    assert len(tbl) == 1
+                    assert type(tbl["a"]) == DataFrame
+            except ValueError:
+                print("Context error")
```

Comment:
remove the "try:" on L96 and the "except ValueError: print(...)" on L100-101, so dedent L97-99

samef for L91, L94-95, and L92-93

L90 change `print("Context error")` to `pass`

we can discuss jreback's suggestion above after these are fixed